packages/clerk-js/src/ui/hooks/useNavigateToFlowStart.ts

```diff
@@ -2,8 +2,14 @@ import { useRouter } from '../router';
 
 export const useNavigateToFlowStart = () => {
   const router = useRouter();
+  const userProfileModalFullPath = '/CLERK-ROUTER/VIRTUAL/user';
```

Comment:
I think you can use the `VIRTUAL_ROUTER_BASE_PATH` const from `VirtualRouter.tsx`, right?

Reply:
I am not sure because we need the full path here. I could add another constant (e.g. VIRTUAL_ROUTER_USER_PROFILE_PATH = '/CLERK-ROUTER/VIRTUAL/user') in VirtualRouter.tsx if we prefer it.